src/main/kotlin/com/controller/DriverController.kt

```diff
@@ -0,0 +1,36 @@
+package com.controller
+
+import com.dto.ReservationManager
+import com.dto.ReservationRequest
+import com.dto.StartedReservationDto
+import com.dto.StoppedReservationDto
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.math.BigDecimal
+import java.time.LocalDateTime
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/driver/reservation")
+class DriverController(private val reservationManager: ReservationManager) {
+
+    @PostMapping("/start")
```

Comment:
start i stop - czy to dobre nazwy operacji restowych?

Reply:
faktycznie POST /reservation powinien zajmować się tworzeniem nowej rezerwacji, a PUT lub PATCH zakończeniem transakcji bo przecież powinno się opierać URI na zasobach a nie akcjach.